src/System.Management.Automation/engine/parser/token.cs

```diff
@@ -669,7 +675,7 @@ public enum TokenFlags
         SpecialOperator = 0x00001000,
 
         /// <summary>
-        /// The token is one of the assignment operators: '=', '+=', '-=', '*=', '/=', or '%='
+        /// The token is one of the assignment operators: '=', '+=', '-=', '*=', '/=', '%=' or '??='
```

Comment:
I don't see any change in Parser.cs ... At least the comments in `BinaryExpressionRule` should be updated to include `??`. The same goes to `??=`.

Reply:
Added